src/parser.cpp

```diff
@@ -479,6 +479,103 @@ options_metadata parser::load_settings()
         "node.refresh_transactions",
         value<bool>(&configured.node.refresh_transactions),
         "Request transactions on each channel start, defaults to false."
+    )
+
+    /* [bitcoin] */
+    (
+        "bitcoin.retargeting_factor",
+        value<uint32_t>(&configured.bitcoin.retargeting_factor),
+        "The retargeting factor, defaults to 4."
+    )
+    (
+        "bitcoin.block_spacing_seconds",
+        value<uint32_t>(&configured.bitcoin.block_spacing_seconds),
+        "The target block spacing time in seconds, defaults to 600."
+    )
+    (
+        "bitcoin.timestamp_limit_seconds",
+        value<uint32_t>(&configured.bitcoin.timestamp_limit_seconds),
+        "The amount of seconds the timestamp is allowed to drift, defaults to 7200."
+    )
+    (
+        "bitcoin.retargeting_interval_seconds",
+        value<uint32_t>(&configured.bitcoin.retargeting_interval_seconds),
+        "The target timespan for the retargeting interval in seconds, defaults to 1209600."
+    )
+    (
+        "bitcoin.proof_of_work_limit",
+        value<uint32_t>(&configured.bitcoin.proof_of_work_limit),
+        "The proof of work limit, defaults to 486604799."
+    )
+    (
+        "bitcoin.genesis_block",
+        value<config::block>(&configured.bitcoin.genesis_block),
+        "The genesis block."
+    )
+    (
+        "bitcoin.activation_threshold",
+        value<size_t>(&configured.bitcoin.activation_threshold),
+        "The number of new version blocks required for bip34 style soft fork activation, defaults to 750."
+    )
+    (
+        "bitcoin.enforcement_threshold",
+        value<size_t>(&configured.bitcoin.enforcement_threshold),
+        "The number of new version blocks required for bip34 style soft fork enforcement, defaults to 950."
+    )
+    (
+        "bitcoin.activation_sample",
+        value<size_t>(&configured.bitcoin.activation_sample),
+        "The total number of new version blocks considered for bip34 style soft fork, defaults to 1000."
```

Comment:
"The number of blocks considered for bip34 style soft fork activation, defaults to 1000."

Reply:
Done.